eoservices/src/main/java/com/pcsinfotech/eoservices/service/IsoCodesService.java

```diff
@@ -0,0 +1,59 @@
+package com.pcsinfotech.eoservices.service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.util.CollectionUtils;
+import org.springframework.util.StringUtils;
+
+import com.pcsinfotech.eoservices.model.IsoCode;
+import com.pcsinfotech.eodata.entities.*;
+import com.pcsinfotech.eodata.repositories.*;
+
+@Service
+public class IsoCodesService {
+	
+	@Autowired
+	private CountryRepository countryRepository;
+	
+	public List<IsoCode> getIsoCodes() {
+		
+		List<IsoCode> isoCodes = new ArrayList<IsoCode>();
+		
+		List<Country> dbIsoCodes = countryRepository.getIsoCodes();
+		if (!CollectionUtils.isEmpty(dbIsoCodes)) {
+			dbIsoCodes.stream().forEach(t -> {
+					IsoCode code = new IsoCode();
+					code.setCountry(t.getCountry());
+					code.setIsoCode(t.getIsoCode());
+					isoCodes.add(code);
+			});
+		}
+		else {
+			//Do Nothing
```

Comment:
@vgurramkonda-pcs If we are doing nothing in the "else" block, the else block itself can be removed altogether

Reply:
Done.